front/components/vaults/VaultResourcesList.tsx

```diff
@@ -121,6 +124,35 @@ const getTableColumns = ({
     },
   };
 
+  const usedByColumn = {
+    header: "Used by",
+    id: "usedBy",
+    accessorFn: (row: RowData) => row.dataSourceView.usage,
```

Comment:
Do you need the `accessorFn` here? 🤔

Reply:
I'm not 100% sure :) Will check.

Reply:
IIRC you only need it when `{info.getValue()}`